packages/core/src/lib/telemetry.ts

```diff
@@ -285,38 +292,49 @@ export async function runTelemetry (
       return
     }
 
-    const { telemetry } = getDefaultedTelemetryConfig()
+    const { telemetry, userConfig } = getTelemetryConfig()
 
     // don't run if the user has opted out
     //   or if somehow our defaults are problematic, do nothing
-    if (!telemetry) return
+    if (telemetry === false) return
 
     // don't send telemetry before we inform the user, allowing opt-out
-    if (!telemetry.informedAt) return inform()
+    const telemetryDefaulted = getDefault(telemetry)
+    if (!telemetryDefaulted.informedAt) return inform(telemetryDefaulted, userConfig)
```

Comment:
Suggest adding an explicit log here to remind that telemetry is active:

```js
    console.log('🛎️  Telemetry is enabled')
```

![image](https://github.com/user-attachments/assets/b0db2682-ef80-4ecc-b324-ca4be76b34de)

Reply:
This is pretty neat

Reply:
What about _disabled_?

Reply:
```
    console.log('🛎️  Telemetry is enabled')
    // or
    console.log('✅  Telemetry is enabled')
```

and

```
    console.log('❎  Telemetry is disabled')
```

Reply:
At some stage I want to switch to declarative wording here rather than procedural (`Database unchanged` versus `Connecting to the database`)

Reply:
@kennedybaird can we do this in a different pull request?
I want to merge this in ASAP and this might require a new set of wiring and functions

Reply:
And, like in #8118, the phrasing of this might resurface whether we want to try and manage our dependencies telemetry configuration too

Reply:
Sure :)

Reply:
#9299